app/views/components/p1.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(
+  content: Html,
+  id: Option[String] = None,
+  classes: Option[String] = None,
+  link: Option[Html] = None,
+  tabLink: Option[Html] = None
+)(
+  implicit messages: Messages
+)
+
+<p @{id.fold("")(id => s"id=$id")}  class="@{classes.fold("govuk-body")(classes => s"$classes")}">@content @link @tabLink</p>
```

Comment:
Is there any reason why we have another `P` component, can't we not just have one ?

Reply:
as of now it is being crated (like in other repos) to avoid the refactoring in quite a few views. You are right and I looked into this possibility and ideally we should only have one p component. There are 8 views (in this repo) where the existing p component is being used so to refactor these, to add the unit tests and complete the testing of all these views for different scenarios, it will take considerable amount of time and it should be done as part of refactoring as a technical debt. Let me know your thoughts.